### ISSUE TITLE
Fix warnings about 'INSTANTIATE_TYPED_TEST_CASE_P' gtest macro usage

### ISSUE THREAD

**Issue by lifflander** [DARMA-tasking/vt#465]
**Describe the bug**
```
/home/travis/build/DARMA-tasking/vt/tests/unit/memory/test_memory_active.cc:163:74: warning: ISO C++11 requires at least one argument for the "..." in a variadic macro
5176 INSTANTIATE_TYPED_TEST_CASE_P(test_mem_short,  TestMemoryActive, MsgShort);
5177                                                                          ^
5178/home/travis/build/DARMA-tasking/vt/tests/unit/memory/test_memory_active.cc:164:75: warning: ISO C++11 requires at least one argument for the "..." in a variadic macro
5179 INSTANTIATE_TYPED_TEST_CASE_P(test_mem_normal, TestMemoryActive, MsgNormal);
5180                                                                           ^
5181/home/travis/build/DARMA-tasking/vt/tests/unit/memory/test_memory_active.cc:165:75: warning: ISO C++11 requires at least one argument for the "..." in a variadic macro
5182 INSTANTIATE_TYPED_TEST_CASE_P(test_mem_serial, TestMemoryActive, MsgSerial);
5183                                                                           ^
5184
[ 68%] Building CXX object tests/CMakeFiles/test_query_context.dir/unit/collection/test_query_context.cc.o
5249/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_index_types.cc:129:1: warning: ISO C++11 requires at least one argument for the "..." in a variadic macro
5250 );
5251 ^
[ 56%] Linking CXX executable test_memory_lifetime
1391/home/travis/build/DARMA-tasking/vt/tests/unit/memory/test_memory_active.cc:163:74: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1392INSTANTIATE_TYPED_TEST_CASE_P(test_mem_short,  TestMemoryActive, MsgShort);
1393                                                                         ^
1394/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1395# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1396         ^
1397/home/travis/build/DARMA-tasking/vt/tests/unit/memory/test_memory_active.cc:164:75: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1398INSTANTIATE_TYPED_TEST_CASE_P(test_mem_normal, TestMemoryActive, MsgNormal);
1399                                                                          ^
1400/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1401# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1402         ^
1403/home/travis/build/DARMA-tasking/vt/tests/unit/memory/test_memory_active.cc:165:75: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1404INSTANTIATE_TYPED_TEST_CASE_P(test_mem_serial, TestMemoryActive, MsgSerial);
1405                                                                          ^
1406/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1407# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1408         ^
14093 
 74%] Linking CXX executable test_active_bcast_put
1538/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_construct_no_idx.cc:90:1: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1539);
1540^
1541/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1542# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1543         ^
1544/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_construct_no_idx.cc:94:1: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1545);
1546^
1547/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1548# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1549         ^
/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_send.cc:200:1: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1617);
1618^
1619/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1620# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1621         ^
1622/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_send.cc:203:1: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1623);
1624^
1625/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1626# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1627         ^
16282 warnings generated.
1629[ 82%] Linking CXX executable test_send
1630/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_construct.cc:133:1: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1631);
1632^
1633/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1634# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1635         ^
1636/home/travis/build/DARMA-tasking/vt/tests/unit/collection/test_construct.cc:137:1: warning: must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments]
1637);
1638^
1639/home/travis/build/DARMA-tasking/vt/googletest/install/include/gtest/gtest-typed-test.h:290:10: note: macro 'INSTANTIATE_TYPED_TEST_CASE_P' defined here
1640# define INSTANTIATE_TYPED_TEST_CASE_P(Prefix, CaseName, Types, ...)      \
1641         ^
```

**Comment by pnstickne** [comment]
Not a stdlibc++ issue (no issue GCC 9.2 / Clang 8 + Gtest 1.18). gtest 1.18.1 is not affected; appears to be related due to deprecation: https://github.com/google/googletest/releases - INSTANTIATE_TYPED_TEST_CASE_P

**Comment by pnstickne** [comment]
https://github.com/google/googletest/pull/2316 - gtest core issue, should be fixed 1.10.1

**Comment by pnstickne** [comment]
FWIW: dependency minor defect, only warnings, only building tests, only specific compiler / flags, gtest release with fix is not new enough to hit common distributions.

**Comment by lifflander** [comment]
Fixed in VT via PR #512
